templates/python/custom-copilot-rag-custom-api/README.md.tpl

```diff
@@ -17,9 +17,8 @@ This template showcases a bot app that responds to user questions like an AI ass
 > - An account with [OpenAI](https://platform.openai.com/)
 {{/useOpenAI}}
 {{#useAzureOpenAI}}
-> - Prepare your own [Azure OpenAI](https://aka.ms/oai/access) resource.
+> - An account with [Azure OpenAI](https://aka.ms/oai/access).
 {{/useAzureOpenAI}}
-> - [Node.js](https://nodejs.org/) (supported versions: 16, 18) for local debug in Test Tool.
```

Comment:
Since we have testtool support, should we mention this dependency for test tool?

Reply:
Delete it to keep this part same as other templates.